mmv1/third_party/terraform/tests/resource_dataproc_cluster_test.go.erb

```diff
@@ -220,8 +215,8 @@ func TestAccDataprocCluster_basic(t *testing.T) {
 	})
 }
 
-<% if version == "ga" -%>
 func TestAccDataprocVirtualCluster_basic(t *testing.T) {
+	skipIfVcr(t)
```

Comment:
I'd propose we keep running this test in VCR. I understand that the test is failing due to an error unrelated to this PR. We've already got an issue tracking that error and hopefully we'll get the issue fixed shortly. Keep this test in VCR just in case we forget to bring it back after the problem solved. Plus, we'll not be blocked from merging this PR by this unrelated test failure!